src/verification/v1.8.0.ts

```
@@ -253,25 +253,16 @@ export const verifyDVV1X8 = (clusterLock: ClusterLock): boolean => {
 
     // Check deposit amounts match exactly if they are defined
     const depositAmounts = clusterLock.cluster_definition.deposit_amounts;
-    if (depositAmounts !== null) {
+    if ( !!depositAmounts && depositAmounts!== null) {
```

ESLint

[error] There should be no space after this paren.

(space-in-parens)